src/test/java/gov/di_ipv_core/pages/MobileAppPage.java

```diff
@@ -0,0 +1,29 @@
+package gov.di_ipv_core.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class MobileAppPage {
```

Comment:
Can we change the name of this class to refelect that it is a stub.
```suggestion
public class MobileAppStubPage {
```

Reply:
Updated